File: AlgoExpertPy/Arrays/3-Sorted Squared Array.py

def sortedSquaredArray(array):
    idx = 0
    while idx < len(array):
        array[idx] *= array[idx]
        idx += 1

    array.sort()
    return array

array = [1,2,3,4,5,6,7,8]  
print(array)
print(sortedSquaredArray(array))
print(array)

File: AlgoExpertJS/3-Sorted Squared Array.js
function sortedSquaredArray(array) {
    for (let i = 0; i< array.length; i++) {
        array[i] = array[i] * array[i]
    }
    return array.sort(function(a,b) {
        return a - b
    })
}

const result = sortedSquaredArray([-3, 8, 5, 56, 7, -13, -32, 4, -2, 0])
console.log(result)